doc/plutus-core-spec/flat-serialisation.tex

```diff
@@ -195,16 +195,17 @@ \subsection{Arrays}
 
 \subsection{Natural numbers}
 We encode natural numbers by splitting their binary representations into
-sequences of 7-bit blocks, then emitting these as a list with the \textbf{least
-  significant block first}:
+sequences of 7-bit blocks, then emitting all but last chunks as a list with
+the \textbf{least significant block first}. The last chunk(most significant chunk)
```

Comment:
```suggestion
the \textbf{least significant block first}. The last chunk (most significant chunk)
```

Reply:
We're calling things both blocks and chunks.  We should stick to one, probably "blocks" since I think we use "chunk" elsewhere to refer to things whose length may vary.